projects/demo/src/modules/components/editor/embed/html5/examples/1/index.ts

```diff
@@ -85,8 +85,12 @@ import {map} from 'rxjs/operators';
             },
         },
     ],
+    host: {
+        class: 'html5-editor-example',
+        '[class._cypress]': 'isCypress',
+    },
+    encapsulation: ViewEncapsulation.None,
```

Comment:
Maybe it is better to use `::ng-deep` instead ?

Reply:
fixed